org.freedesktop.Sdk.Extension.haskell.json

```diff
@@ -0,0 +1,96 @@
+{
+    "id": "org.freedesktop.Sdk.Extension.haskell",
+    "branch": "19.08",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "19.08",
+    "build-extension": true,
+    "sdk": "org.freedesktop.Sdk",
+    "separate-locales": false,
+    "appstream-compose": false,
+    "build-options": {
+        "prefix": "/usr/lib/sdk/haskell"
+    },
+    "modules": [
+        {
+            "name": "ghc",
+            "buildsystem": "autotools",
+            "make-args": [
+                "install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.haskell.org/~ghc/latest/ghc-8.10.1-x86_64-fedora27-linux.tar.xz",
+                    "sha512": "12e3c6816fb1306bd8450fd9a0463967de59ccf24190e76d12aa659e35deb404a4a809f5b6369ffba8771b9f26a3358c19db4261b0f0ab4d52a066f16e68ff1b"
+                }
+            ]
+        },
+        {
+            "name": "cabal",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.haskell.org/~cabal/cabal-install-3.2.0.0/cabal-install-3.2.0.0-x86_64-unknown-linux.tar.xz",
```

Comment:
Should use flatpak-external-data-checker for detecting when updates are available.